L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc

```diff
@@ -610,8 +612,24 @@ void L1FPGATrackProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSe
 
   const std::vector<trklet::Track>& tracks = eventProcessor.tracks();
 
+  // number of track channels
+  const unsigned int numStreamsTrack = N_SECTOR * channelAssignment_->numChannelsTrack();
+  // number of stub channels
+  const unsigned int numStreamsStub = N_SECTOR * channelAssignment_->numChannelsStub();
+  // number of stub channels if all seed types streams padded to have same number of stub channels (for coding simplicity)
+  unsigned int maxNumProjectionLayers = 0;
+  for (int iSeedType = 0; iSeedType < channelAssignment_->numSeedTypes(); iSeedType++) {
+    maxNumProjectionLayers =
+        max(maxNumProjectionLayers, (unsigned int)channelAssignment_->numProjectionLayers(iSeedType));
+  }
+  const unsigned int numStreamsStubRaw = numStreamsTrack * maxNumProjectionLayers;
+
+  // Streams formatted to allow this code to run outside CMSSW.
+  vector<vector<string>> streamsTrackRaw(numStreamsTrack);
+  vector<vector<StubStreamData>> streamsStubRaw(numStreamsStubRaw);
```

Comment:
I see that you are not reserving the inner vector before filling, please do that or use deques